#goal definition
std_msgs/String folder_path
std_msgs/String calib_path
int32 id_marker
---
#result definition
bool finished
---
#feedback
int32 n_imgs

